BringgSDK.js

```diff
@@ -937,7 +940,7 @@ var BringgSDK = (function () {
   module._setDriverActivity = function (newActivity) {
     driverActivity = google.maps.TravelMode.DRIVING;
 
-    switch (newActivity) {
+    switch (parseInt(newActivity)) {
```

Comment:
i think the parseInt should be external to this. i.e you expect to receive an integer/enum, so let your clients to pass you the right format. 
otherwise if you want to handle them passing arbitrary values, you need a better parsing error handling...

Reply:
fixed